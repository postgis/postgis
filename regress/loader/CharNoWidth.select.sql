SET CLIENT_ENCODING to UTF8;
SELECT * FROM loadedshp;
SELECT column_name, character_maximum_length FROM information_schema.columns WHERE table_name = 'loadedshp' ORDER BY column_name;
