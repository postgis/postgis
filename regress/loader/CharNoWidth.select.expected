1|11|50000|02390665|1150000|Washington|Washington city|25|C5|||G4110|N|158316184|18709787|+38.9042474|-077.0165167
aland|
awater|
classfp|2
funcstat|1
geoid|7
gid|
intptlat|11
intptlon|12
lsad|2
mtfcc|5
name|100
namelsad|100
pcicbsa|
pcinecta|
placefp|5
placens|8
statefp|2
